akka-cluster-sharding-typed/src/test/java/akka/cluster/sharding/typed/ActiveActiveShardingTest.java

```diff
@@ -144,7 +144,9 @@ public ForwardToAll(String entityId, MyActiveActiveStringSet.Command message) {
                 Arrays.asList(
                     new ReplicaId("DC-A"), new ReplicaId("DC-B"), new ReplicaId("DC-C"))));
 
-    private final ActiveActiveSharding<MyActiveActiveStringSet.Command> aaSharding;
+    private final ActiveActiveSharding<
+            MyActiveActiveStringSet.Command, ShardingEnvelope<MyActiveActiveStringSet.Command>>
+        aaSharding;
```

Comment:
And here is it back. 😞 

I was slightly happy to have seen it gone.

Reply:
Watch it or I'll add yet another type parameter!